src/duration.js

```diff
@@ -172,7 +173,18 @@ export default class Duration {
    * @private
    */
   constructor(config) {
+    if (config.conversionAccuracy && config.matrix)
+      throw new Error(
```

Comment:
Not sure if it's ok for you to throw error, but since I couldn't drop the `config.conversionAccuracy` config for obvious reasons, I decided to add a safety guard.

Reply:
It's ok to throw an error here, but the errors need to be typed. See https://github.com/moment/luxon/blob/master/src/errors.js